kolibri/plugins/example_plugin/kolibri_plugin.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+"""
+Kolibri example plugin
+======================
+
+Copy and modify this code for your own plugin.
+
+"""
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+
+from django.utils.translation import ugettext as _
+
+from kolibri.plugins.base import KolibriPluginBase
+from kolibri.plugins.hooks import NAVIGATION_POPULATE
+
+
+class ExamplePlugin(KolibriPluginBase):
+
+    @classmethod
+    def enable(cls, config):
+        print("Activating example plugin")
+        # Make this automatic and use __name__ ?
+        config["INSTALLED_APPS"].append("kolibri.plugins.example_plugin")
```

Comment:
By the way, this will happily append the same app twice to INSTALLED_APPS, resulting in an error.